modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -0,0 +1,89 @@
+
+An output can be either:
+
+* `elasticsearch` to forward logs to an external Elasticsearch cluster, specified by server name or FQDN.
+* `forward` to forward logs to an external log aggregation solution. This option uses the Fluentd `forward` protocol.
+
+A log source is the type of data you can forward:
+
+* `logs.app` - Container logs generated by user applications running in the cluster, except infrastructure container applications.
+* `logs.infra` - Logs generated by both infrastructure components running in the cluster and {product-title} nodes, such as journal logs. Infra components are applications run on pods in the `openshift*`, `kube*`, or `default` projects.
+* `logs.audit` - Logs generated by the node Audit system (auditd), which are stored in the  */var/log/audit/audit.log* file, and the audit logs from the Kubernetes apiserver and the OpenShift apiserver. 
+
+Note the following:
+
+* Log forwarding does not provide secure storage for audit logs. You are responsible to ensure the endpoint is compliant with your organizational and governmental regulations and is properly secured. {product-title} cluster logging does not comply with those regulations.
+
+* You are responsible to create and maintain any additional configurations that external destinations might require, such as keys and secrets, service accounts, port opening, or global proxy configuration.
+
+* An output can support TLS communication using secrets. Secrets must have keys of: *tls.crt*, *tls.key*, and *ca-bundler.crt* which point to the respective certificates for which they represent. Secrets shall have the key *shared_key* for use when using forward in a secure manner.
+
+.Sample log forwarding outputs and pipelines
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+spec:
+  disableDefaultForwarding: true <1>
+  outputs: <2>
+   - type: "elasticsearch" <3>
+     name: elasticsearch <4>
+     endpoint: elasticsearch.svc.messaging.cluster.local <5>
+     secrets: <6>
+        name: elasticsearch
+   - type: "elasticsearch"
+     name: elasticsearch-insecure
+     endpoint: elasticsearch-insecure.svc.messaging.cluster.local
+     insecure: true <7>
+   - type: "forward"
+     name: secureforward-offcluster
+     endpoint: https://secureforward.offcluster.com:9200
+     secrets:
+        name: secureforward
+  pipelines: <8>
+   - name: container-logs <9>
+     inputSource: logs.app <10>
+     outputRefs: <11>
+     - elasticsearch
+     - secureforward-offcluster
+   - name: infra-logs
+     inputSource: logs.infra
+     outputRefs:
+     - elasticsearch-insecure
+   - name: audit-logs
+     inputSource: logs.audit
+     outputRefs:
+     - secureforward-offcluster
+----
+<1> Parameter to enable log forwarding. Set to `true` to enable.
```

Comment:
this disables  CLO from creating a default version of LogForwarding when set to true